packages/grafana-ui/src/components/Select/SelectBase.tsx

```diff
@@ -342,6 +342,7 @@ export function SelectBase<T>({
           }),
           container: () => ({
             position: 'relative',
+            paddingRight: '4px',
```

Comment:
This change would be applied to all Select components across whole Grafana, which is probably not what we wanted. Instead of that, you can pass className prop to your Selct component and add padding/margin there. Also, we ideally don't want to hard-code number of pixels and instead of that use theme - something like `${theme.spacing(0.5)}`.

```
 <Select
                width={16}
                isSearchable={false}
                options={STEP_OPTIONS}
                onChange={this.onStepChange}
                value={stepOption}
```